tests/api.rs

```diff
@@ -139,15 +147,17 @@ async fn test_tenants_and_users() {
                 .unwrap();
 
             // Verify that the API has roundtripped the key properties.
-            assert_eq!(created_user.name, name);
+            // create_user.name is returning an email
+            // https://github.com/MaterializeInc/rust-frontegg/issues/11
+            // assert_eq!(created_user.name, name);
             assert_eq!(created_user.email, email);
 
             // Verify that fetching the same user by ID from the API returns
             // the same properties.
             let user = client.get_user(created_user.id).await.unwrap();
             assert_eq!(created_user.id, user.id);
-            assert_eq!(created_user.name, user.name);
-            assert_eq!(created_user.email, user.email);
+            assert_eq!(user.name, name);
+            assert_eq!(user.email, email);
```

Comment:
We definitely want to make sure the user name is the name we set during create here, rather than the CreateUser object they give us, which is unfortunately wrong at the moment.